lit_tests/kernel/wave/index_sequence_analysis.py

```diff
@@ -198,13 +196,13 @@ def test_gemm():
         # CHECK-NEXT: placeholder(_name=b
         # CHECK-NEXT: placeholder(_name=c
         # CHECK-NEXT: register
-        # CHECK-SAME: index={M: $WG0*BLOCK_M + 4*floor((Mod($T0, 64))/16) : 4 : 16, N: $WG1*BLOCK_N + BLOCK_N/2 + Mod($T0, 16) : 1 : 1})
```

Comment:
Can we double check why we need to change the LIT test at all? If the implicitWaveConstraint is indeed 1:1 functionality with the WaveConstraint, we shouldn't be changing these